distribution/packages/build.gradle

```diff
@@ -54,9 +54,7 @@ buildscript {
   // Be aware that it seems the redline project hasnt been active for a while
   configurations.all {
     resolutionStrategy {
-        dependencySubstitution {
-          substitute module('org.redline-rpm:redline') using module('com.github.breskeby:redline:9c85270')
-        }
+      force 'org.redline-rpm:redline:1.2.10'
```

Comment:
How come we force it? Does Gradle ship its own, or something like that?

Reply:
what that means is that we enforce a particular version (1.2.10). Otherwise redline 1.2.9 would be brought in as a transitive dependency by the nebula.ospackage plugin that has 1.2.9 declared. (There's an open PR on ospackage to fix this).